src/client/cl_cgame.c

```diff
@@ -127,7 +129,7 @@ void Cl_InitCgame(void) {
 	import.Warn_ = Com_Warn_;
 	import.Error_ = Cl_CgameError;
 
-	import.Time = Sys_Milliseconds;
+	import.Time = SDL_GetTicks;
 
```

Comment:
Hm. I'm glad you pointed this one out in the changeset. Rather than expose an SDL library function to the cgame, I think we can remove this pointer. The cgame can already access this value through `cgi.client->systime`.